src/number_theory/padics/padic_integers.lean

```diff
@@ -192,79 +192,69 @@ instance is_absolute_value : is_absolute_value (λ z : ℤ_[p], ∥z∥) :=
 { abv_nonneg := norm_nonneg,
   abv_eq_zero := λ ⟨_, _⟩, by simp [norm_eq_zero],
   abv_add := λ ⟨_,_⟩ ⟨_, _⟩, norm_add_le _ _,
-  abv_mul := λ _ _, by simp only [norm_def, padic_norm_e.mul, padic_int.coe_mul]}
+  abv_mul := λ _ _, by simp only [norm_def, padic_norm_e.mul, padic_int.coe_mul] }
 
 variables {p}
 
-instance : is_domain ℤ_[p] :=
-function.injective.is_domain (subring p).subtype subtype.coe_injective
+instance : is_domain ℤ_[p] := function.injective.is_domain (subring p).subtype subtype.coe_injective
 
 end padic_int
 
 namespace padic_int
+
 /-! ### Norm -/
+
 variables {p : ℕ} [fact p.prime]
 
 lemma norm_le_one (z : ℤ_[p]) : ∥z∥ ≤ 1 := z.2
 
-@[simp] lemma norm_mul (z1 z2 : ℤ_[p]) : ∥z1 * z2∥ = ∥z1∥ * ∥z2∥ :=
-by simp [norm_def]
+@[simp] lemma norm_mul (z1 z2 : ℤ_[p]) : ∥z1 * z2∥ = ∥z1∥ * ∥z2∥ := by simp [norm_def]
 
-@[simp] lemma norm_pow (z : ℤ_[p]) : ∀ n : ℕ, ∥z^n∥ = ∥z∥^n
+@[simp] lemma norm_pow (z : ℤ_[p]) : ∀ n : ℕ, ∥z ^ n∥ = ∥z∥ ^ n
```

Comment:
I like this better than the alternative, but again, we should probably formalize this in the style guide.

Reply:
I think this is formalized by the fact that the notation for `^` inserts spaces in the pretty-printer.

Reply:
Sorry I'm not sure which is preferred - should I do `z^n` or `z ^ n`?

Reply:
The latter.